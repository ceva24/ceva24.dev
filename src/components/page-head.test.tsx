import React from "react";
import { render } from "@testing-library/react";
import PageHead from "./page-head";

describe("page-head", () => {
    it("renders", () => {
        const { asFragment } = render(<PageHead />);

        expect(asFragment()).toMatchSnapshot();
    });

    it("renders with a title", () => {
        const { asFragment } = render(<PageHead title="Test title" />);

        expect(asFragment()).toMatchSnapshot();
    });
});
