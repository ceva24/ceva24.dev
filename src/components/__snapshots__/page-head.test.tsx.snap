// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`page-head renders 1`] = `
<DocumentFragment>
  <meta
    content="Chris Evans, Senior Engineering Manager at the LEGO Group, UK"
    name="description"
  />
  <title>
    Chris Evans
  </title>
</DocumentFragment>
`;

exports[`page-head renders with a title 1`] = `
<DocumentFragment>
  <meta
    content="Chris Evans, Senior Engineering Manager at the LEGO Group, UK"
    name="description"
  />
  <title>
    Test title
  </title>
</DocumentFragment>
`;
